map_claims_test.go

```diff
@@ -106,6 +106,15 @@ func TestMapclaimsVerifyExpiresAtInvalidTypeString(t *testing.T) {
 	}
 }
 
+func TestMapclaimsVerifyExpiresRequiredButMissing(t *testing.T) {
```

Comment:
I would suggest du integrate this into `jwtTestData` in parser_test.go instead. We want to get rid of the single test functions at some point

Reply:
done